src/v2/view-builder/views/ov/EnrollPollOktaVerifyView.js

```diff
@@ -45,33 +45,17 @@ const Body = BaseForm.extend(Object.assign(
       }
     },
     showMessages() {
```

Comment:
same question here, this seems to be a good use case for `showCustomFormErrorCallout` instead